citrixitm/resource_citrixitm_dns_app_test.go

```diff
@@ -19,9 +19,21 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+const minimalAppSource = `function init(config) {}
+
+function onRequest(request, response) {
+    response.addCName('foo.example.com');
+    response.setTTL(20);
+}
+`
+
+const updatedMinimalAppSource = minimalAppSource + "\n// foo"
```

Comment:
does `updateMinimalAppSource` try to capture what would be returned by the portal API?

Reply:
`updateMinimalAppSource` represents a _change_ to an existing app's JS that will be submitted via Terraform. In this case, it just adds a comment on a new line.

Reply:
@aggarwaa Oh, yes, that's right. It's the "expected" value that is returned by the API _before_ any leading/trailing whitespace has been removed. It's also used to prepare the app code to be submitted _to_ the API by escaping "\n" to "\\n".